scripts/run.sh

```diff
@@ -241,6 +241,8 @@ run_simulator() {
   #################################
   echo "building simulator"
   pushd ./cmd/simulator
+  # to use the latest subnet-evm from the local repository
+  go mod edit -replace github.com/ava-labs/subnet-evm=../..
```

Comment:
💯

Reply:
Can we include this directly in the `go.mod` file of the simulator?

Reply:
That was my initial go Mod file lol. I can revert back to replace syntax in go module file.